cli/coverage.rs

```diff
@@ -322,14 +318,18 @@ fn new_box_with<T>(new_fn: impl FnOnce(*mut T) -> T) -> Box<T> {
 }
 
 pub fn filter_script_coverages(
-  coverages: Vec<ScriptCoverage>,
+  coverages: Vec<Coverage>,
   test_file_url: Url,
   test_modules: Vec<Url>,
-) -> Vec<ScriptCoverage> {
+) -> Vec<Coverage> {
   coverages
     .into_iter()
     .filter(|e| {
-      if let Ok(url) = Url::parse(&e.url) {
+      if let Ok(url) = Url::parse(&e.script_coverage.url) {
+        if url.path().ends_with("__anonymous__") {
```

Comment:
Since anonymous scripts are now inspectable, ignore it to keep the semantics the same.